app/test/frontend/golden/landing_page.html

```diff
@@ -87,9 +87,8 @@ <h2 class="_visuallyhidden">Dart package manager</h2>
           </div>
         </div>
         <p class="text">Find and use packages to build 
-          <a href="/flutter">Flutter</a> and 
-          <a href="/web">web</a> apps with 
-          <a target="_blank" rel="noopener" href="https://dart.dev">Dart</a>.
+          <a href="/dart">Dart</a> and 
+          <a href="/flutter">Flutter</a> and apps.
```

Comment:
Please remove extra 'and' before apps.

Currently the full sentence is:
_...build Dart and Flutter and apps._

It should be:
_...build Dart and Flutter apps._